src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1087,6 +1087,17 @@ public override TypeInfo GetTypeInfo(SelectOrGroupClauseSyntax node, Cancellatio
         private void GetBoundNodes(CSharpSyntaxNode node, out CSharpSyntaxNode bindableNode, out BoundNode lowestBoundNode, out BoundNode highestBoundNode, out BoundNode boundParent)
         {
             bindableNode = this.GetBindableSyntaxNode(node);
+
```

Comment:
added this to make test pass. it looks like GetBindableSyntaxNode can return node that is outside of Root node for this member semantic model